.gitignore

```diff
@@ -20,6 +20,9 @@ configure
 src/include/BuildConfig.h.in
 src/include/BuildConfig.h.in~
 
+# GN build system
```

Comment:
Can we use /build/ here to align with the existing auto tools stuff? (also vscode, etc)